<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsObjectDialog</class>
 <widget class="QDialog" name="GraphicsObjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="objectTypeLabel">
         <property name="text">
          <string>Object Type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="objectTypeField"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="signalNameLabel">
         <property name="text">
          <string>Signal Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="signalNameLineEdit"/>
         </item>
         <item>
          <widget class="QCheckBox" name="showSignalName">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Show</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="aliasLabel">
         <property name="text">
          <string>Alias</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="aliasLineEdit"/>
         </item>
         <item>
          <widget class="QCheckBox" name="showAlias">
           <property name="text">
            <string>Show</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="showPhysicalValue">
         <property name="text">
          <string>Show Physical Value</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="rawValueLineEdit">
         <property name="text">
          <string>Show Raw Value</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="rawValueComboBox"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="sizeLabel">
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="sizeSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="angleLabel">
         <property name="text">
          <string>Angle</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="angleSpinBox">
         <property name="maximum">
          <number>360</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="colorLabel">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="colorButton">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QSpinBox" name="heightSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="heighLabel">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSpinBox" name="widthSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="widthLabel">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QComboBox" name="orientationComboBox"/>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="orientationLabel">
         <property name="text">
          <string>Orientation</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="minColorLabel">
         <property name="text">
          <string>Min Color</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="maxColorLabel">
         <property name="text">
          <string>Max color</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="minColorButton">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="maxColorButton">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="minValueLabel">
         <property name="text">
          <string>Min Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="minValueSpinbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>-10000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="maxValueLabel">
         <property name="text">
          <string>Max Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="maxValueSpinbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>-10000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="previewLabel">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsPreview"/>
     </item>
     <item>
      <widget class="QSlider" name="signalValueSlider">
       <property name="minimum">
        <number>-10</number>
       </property>
       <property name="maximum">
        <number>110</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GraphicsObjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GraphicsObjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
